contrib/python-bindings/tests/manifold_wrapper.py

```diff
@@ -67,5 +67,47 @@ def test_manifold(self):
 
         self.assertTrue(abs(volume - 4./3. * math.pi) / (4./3.*math.pi) < 2e-2)
 
+class TestManifoldWrapperFunction(unittest.TestCase):
+
+    def setUp(self):
+        self.triangulation = Triangulation('2D')
+        p_center = Point([0., 0., 0.])
+        self.triangulation.generate_hyper_cube()
+
+        self.manifold_1 = Manifold(dim = 2, spacedim = 2)
+        self.manifold_1.create_function("x^2;y^2", "sqrt(x);sqrt(y)")
+
+        self.manifold_2 = Manifold(dim = 2, spacedim = 2)
+        self.manifold_2.create_function(lambda p: [p[0]**2., p[1]**2.],\
+                                        lambda p: [math.sqrt(p[0]), math.sqrt(p[1])] )
+
+    def test_manifold_str(self):
+        self.triangulation.reset_manifold(number = 0)
+        self.triangulation.set_manifold(number = 0, manifold = self.manifold_2)
+        for cell in self.triangulation.active_cells():
+            cell.set_all_manifold_ids(0)
+
+        self.triangulation.refine_global(2)
+
+        volume = 0
+        for cell in self.triangulation.active_cells():
+            volume += cell.measure()
+
+        self.assertTrue(abs(volume - 1.) < 1e-8)
```

Comment:
Won't this test pass even if you don't refine? Would it be possible to have something like https://github.com/dealii/dealii/blob/master/tests/manifold/function_manifold_02.cc and compare with an output file?